### DISCUSSION TITLE
"Reset Global Value to Default Button"

### DISCUSSION THREAD

**Discussion by tylerbuck1** [Security-Onion-Solutions/securityonion#12971] [Polls]
This button seems to be a glaring red flag, that looks like a "back" button just ready to be clicked getting rid of ALL of the current set values at once. Can it be hidden or even be placed in an "advanced mode" setting? or redeveloped all together to make it not so... "clickable".
![RGVDbutton](https://github.com/Security-Onion-Solutions/securityonion/assets/47946889/faa86094-7dde-401d-9629-fbe331a981ee)

**Comment by dougburks** [comment]
Our upcoming 2.4.70 version already includes a fix for this. Clicking that button will ask for confirmation.
